src/docks/playlistdock.ui

```diff
@@ -127,6 +134,13 @@ p, li { white-space: pre-wrap; }
           </property>
          </widget>
         </item>
+        <item>
+         <widget class="QPushButton" name="addFilesToPlaylistButton">
+          <property name="text">
+           <string>Add files to playlist</string>
```

Comment:
The button should not be added twice - once to each page. Rather, the new icon button should be added to the existing row of icons - perhaps between the minus and checkmark - within `horizontalLayout`.

Reply:
OK